### PR TITLE
Change description of Room queries to Dao classname

### PR DESCRIPTION
**Pull request by romtsn**
## :scroll: Description

Change the Room queries description to fully-qualified class name of the Dao

## :bulb: Motivation and Context

image

## :green_heart: How did you test it?
Ran existing tests

## :pencil: Checklist

- [x] I reviewed the submitted code
- [ ] I added tests to verify the changes
- [ ] I updated the docs if needed
- [x] No breaking changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * Fix: Correctly add the proguard UUID output directory to the source set (#226)
 * Feature: Make the ignoreXXX properties in SentryPluginExtension sets (#225)
 * Expose SentryPluginExtension.tracingInstrumentation (#229)
+* Ref: Change Room queries description to Dao class name (#231)
```
Comment (romtsn) [inline CHANGELOG.md:8]:
```suggestion
* Ref: Change Room queries description to Dao class name (#232)
```

File: plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/androidx/room/visitor/AbstractRoomVisitor.kt [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package io.sentry.android.gradle.instrumentation.androidx.room.visitor
+
+import io.sentry.android.gradle.instrumentation.AbstractSpanAddingMethodVisitor
+import org.objectweb.asm.Label
+import org.objectweb.asm.MethodVisitor
+
+abstract class AbstractRoomVisitor(
```
Comment (marandaneto) [inline plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/androidx/room/visitor/AbstractRoomVisitor.kt:7]: this is only DRY right? same code as before + `className`

Comment (romtsn) [reply plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/androidx/room/visitor/AbstractRoomVisitor.kt:7]: yep

**Comment by marandaneto** [comment]
is it possible to unit or integration test it? since no tests were broken its already kinda of a red flag

**Comment by romtsn** [comment]
We'd need to implement https://github.com/getsentry/sentry-android-gradle-plugin/issues/221 first. I know it's not ideal now, but the fact that JVM bytecode verifier accepts the instrumented classes is already a good sign :) and I kinda hand-tested it as you can see on the screenshot. I will prioritize that issue after the file i/o is done

**Review by marandaneto — APPROVED** [review]
LGTM

**Pull request merged**